code/game/objects/items/storage/belt.dm

```diff
@@ -533,7 +533,7 @@
 	STR.can_hold = typecacheof(list(
 		/obj/item/ammo_casing/shotgun
 		))
-
+/* Disabled due to change to holsters to be sloted in pockets instead of belt slot

```

Comment:
`/* Sunset - Disabled due to change to holsters to be sloted in pockets instead of belt slot`

Reply:
Also updated this as well